tests/test_cli.py

```diff
@@ -117,72 +156,170 @@ def test_run(capsys):
+    assert np.allclose(image_data_out['embedding'], image_data_reg['embedding'],
+                       rtol=1e-05, atol=1e-05, equal_nan=False)
 
     # SECOND regression test
-    run(CHIRP_44K_PATH, output_dir=tempdir, suffix='linear', input_repr='linear',
-        content_type='env', embedding_size=512, center=False, hop_size=0.5,
+    run('audio', CHIRP_44K_PATH, output_dir=tempdir, suffix='linear', input_repr='linear',
+        content_type='env', audio_embedding_size=512, audio_center=False, audio_hop_size=0.5,
         verbose=False)
 
     # check output file created
-    outfile = os.path.join(tempdir, 'chirp_44k_linear.npz')
-    assert os.path.isfile(outfile)
+    audio_outfile = os.path.join(tempdir, 'chirp_44k_linear.npz')
+    assert os.path.isfile(audio_outfile)
+
+    run('image', DAISY_PATH, output_dir=tempdir, suffix='linear', input_repr='linear',
+        content_type='env', image_embedding_size=512, verbose=False)
+
+    # check output file created
+    image_outfile = os.path.join(tempdir, 'daisy_linear.npz')
+    assert os.path.isfile(image_outfile)
 
     # regression test
-    data_reg = np.load(REG_CHIRP_44K_LINEAR_PATH)
-    data_out = np.load(outfile)
+    audio_data_reg = np.load(REG_CHIRP_44K_LINEAR_PATH)
+    audio_data_out = np.load(audio_outfile)
+    image_data_reg = np.load(REG_DAISY_LINEAR_PATH)
+    image_data_out = np.load(image_outfile)
 
-    assert sorted(data_out.files) == sorted(data_out.files) == sorted(
+    assert sorted(audio_data_out.files) == sorted(audio_data_reg.files) == sorted(
         ['embedding', 'timestamps'])
-    assert np.allclose(data_out['timestamps'], data_reg['timestamps'],
-                       rtol=1e-05, atol=1e-06, equal_nan=False)
-    assert np.allclose(data_out['embedding'], data_reg['embedding'],
-                       rtol=1e-05, atol=1e-06, equal_nan=False)
+    assert np.allclose(audio_data_out['timestamps'], audio_data_reg['timestamps'],
+                       rtol=1e-05, atol=1e-05, equal_nan=False)
+    assert np.allclose(audio_data_out['embedding'], audio_data_reg['embedding'],
+                       rtol=1e-05, atol=1e-05, equal_nan=False)
+
+    assert sorted(image_data_out.files) == sorted(image_data_reg.files) == ['embedding']
+    assert np.allclose(image_data_out['embedding'], image_data_reg['embedding'],
+                       rtol=1e-05, atol=1e-05, equal_nan=False)
+
+
+    ## Video processing regression tests
+    run('video', BENTO_PATH, output_dir=tempdir, verbose=True)
+
+    # check output files created
+    audio_outfile = os.path.join(tempdir, 'bento_audio.npz')
+    assert os.path.isfile(audio_outfile)
+    image_outfile = os.path.join(tempdir, 'bento_image.npz')
+    assert os.path.isfile(image_outfile)
+
+    # regression test
+    audio_data_reg = np.load(REG_BENTO_AUDIO_PATH)
+    audio_data_out = np.load(audio_outfile)
+    image_data_reg = np.load(REG_BENTO_IMAGE_PATH)
+    image_data_out = np.load(image_outfile)
```

Comment:
Aren't we missing the same for video as well?

Reply:
This is the video test